modules/aioseop_opengraph.php

```diff
@@ -364,7 +363,6 @@ public function init() {
 					'name'       => __( 'Home Description', 'all-in-one-seo-pack' ),
 					'default'    => '',
 					'type'       => 'textarea',
-					'class'      => 'aioseop_count_chars',
```

Comment:
I just realized the operation for adding the class occurs at `admin/aioseop_module_class.php:2661` [link](https://github.com/semperfiwebdesign/all-in-one-seo-pack/blob/a9f2bb6b607c6b6720ff6d702acd0628b6119304/admin/aioseop_module_class.php#L2296), and it was just something I hacked up at the time to get it running and it's based on the 'count' value existing.

Do you think it would be more simplified to keep the class in the setting's configs, rather than having to search where it is being added?

Reply:
@EkoJR That class is also used for the Home Description field on the General Settings screen which presumably uses the same code you link to.  If that's the case then we should leave as is for now, but you may want to open an issue to review and improve this.

Reply:
Actually, that looks like a commit Ashish made...I was thinking it may have been me https://github.com/semperfiwebdesign/all-in-one-seo-pack/pull/1412. At first, I was wondering why it was still working when it was removed, and I believe Ashish might have been unsure where to add the CSS class

Well, I did intend for it to be added to the default_options configs. Since that's where most of the layout configs are, which makes it easier to locate, and allows more control (in case there is a setting where we need the CSS/JS changed or removed from it).

Reply:
@EkoJR I think we can leave as is for now then.